prow/cmd/deck/static/script.js

```diff
@@ -167,7 +169,7 @@ function redraw() {
             r.appendChild(createTextCell(""));
             r.appendChild(createTextCell(""));
         }
-        if (build.url === "" || build.type === "periodic") {
+        if (build.url === "") {
```

Comment:
I don't think any of the periodic jobs have correct build.urls currently. Let's fix that first! One has:

    "url": "https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/_/ci-kubernetes-e2e-gci-gke-reboot-release-1-7/338/",